tests/lib/rules/no-global-assertions.js

```diff
@@ -49,6 +50,7 @@ ruleTester.run("no-global-assertions", rule, {
         wrap("assert.notPropEqual(a, b);"),
         wrap("assert.raises(function () {}, TypeError);"),
         wrap("assert.throws(function () {}, TypeError);"),
+        wrap("assert.true(foo);"),
```

Comment:
I was not able to add `invalid` test cases to this rule for the boolean assertions. Maybe `true` and `false`can't be added as globals because they are reserved keywords in JS?